openpype/tools/tray/pype_tray.py

```diff
@@ -778,6 +778,14 @@ def main():
     if not app:
         app = QtWidgets.QApplication([])
 
+    for attr_name in (
+        "AA_EnableHighDpiScaling",
+        "AA_UseHighDpiPixmaps"
+    ):
+        attr = getattr(QtCore.Qt, attr_name, None)
+        if attr is not None:
+            app.setAttribute(attr)
```

Comment:
If this is only for Pype Tray - don't we know what Qt version would be set up in the environment for that and whether it should be available?
Also, should this log a debug message/warning if an attribute does not exist so we can identify changes of it in the future?

Reply:
> If this is only for Pype Tray - don't we know what Qt version would be set up in the environment for that and whether it should be available?
Also, should this log a debug message/warning if an attribute does not exist so we can identify changes of it in the future?

I've added it as "secure" check for MacOS or linux distributions where it may not be available, depending on installed Qt build. It's a "fancy" feature, definetely should not be breaking. Logs are added, but I can assure you that no one will notice a log message, but everyone will notice pixelated images :)

Reply:
Good point - nice work!